src/app/(pages)/authors/[slug]/page.tsx

```diff
@@ -17,11 +20,13 @@ export default async function ContributorPage({ params: { slug } }) {
 
   return (
     <div>
-      <div style={{ background: 'purple' }}>
-        <BackButton />
+      <div className={styles.container}>
+        <BackButton className={styles.backButton} />
         <AuthorSummary author={author} />
+        <div></div>
       </div>
       <AuthorContentGrid content={contentFromAuthor} />
```

Comment:
I'm getting typescript errors that `contentFromAuthor` doesn't match the `AuthorContentGrid` prop types

Reply:
i believe [35615ea](https://github.com/subvisual/content-sub/pull/32/commits/35615eadf81a73b19270eb7c6f200545e5756ce1) will fix it, please let me know